blocks/v2-hero/v2-hero.css

```diff
@@ -26,24 +26,25 @@
   --scroll-dot-width: 9px;
   --v2-hero-padding: 32px;
 
-  height: calc(100vh - var(--nav-height));
+  min-height: calc(100vh - var(--nav-height));
   overflow: hidden;
   position: relative;
 }
 
 /* Reduce the Hero height by the inpage-navigation height */
 .v2-inpage-navigation-wrapper + .v2-hero-container .v2-hero {
-  height: calc(100vh - (var(--nav-height) + var(--inpage-navigation-height) - 2px));
+  min-height: calc(100vh - (var(--nav-height) + var(--inpage-navigation-height) - 2px));
 }
 
 @supports (height: 1svh) {
   .v2-hero {
-    height: calc(100svh - var(--nav-height));
+    min-height: calc(100svh - var(--nav-height));
   }
 
-  .v2-inpage-navigation-wrapper + .v2-hero-container .v2-hero {
-    height: calc(100svh - (var(--nav-height) + var(--inpage-navigation-height) - 2px));
-  }
+  .v2-inpage-navigation-wrapper + .v2-hero-container .v2-hero, .v2-inpage-navigation-wrapper + .v2-hero-container .v2-hero__content-wrapper {
```

Comment:
Can you put them in 2 different line , each selector classes in one line, for better readability